x-pack/plugins/endpoint/server/services/endpoint/endpoint_query_builders.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { KibanaRequest } from 'kibana/server';
+import { EndpointAppConstants, EndpointAppContext } from '../../types';
+
+export class AllEndpointsQueryBuilder {
+  private readonly request: KibanaRequest<any, any, any>;
+  private readonly endpointAppContext: EndpointAppContext;
+
+  constructor(request: KibanaRequest<any, any, any>, endpointAppContext: EndpointAppContext) {
```

Comment:
Nit: I think this could be shortened to:
```
constructor(private readonly request: KibanaRequest<any, any, any>, private readonly endpointAppContext: EndpointAppContext) {}
```

and you could remove these lines:
```
  private readonly request: KibanaRequest<any, any, any>;
  private readonly endpointAppContext: EndpointAppContext;
```